src/main/java/org/jenkinsci/plugins/github/extension/GHEventsSubscriber.java

```diff
@@ -73,6 +73,9 @@ protected void onEvent(GHEvent event, String payload) {
         return new Function<GHEventsSubscriber, Set<GHEvent>>() {
             @Override
             public Set<GHEvent> apply(GHEventsSubscriber subscriber) {
+                if (subscriber == null) {
```

Comment:
It may happen - see Guava Javadocs. API explicitly allows to pass nulls. If it is not supported, NPE should be thrown